frontend/tests/util/realm.ts

```diff
@@ -27,22 +27,33 @@ const createUserRealm = async (page: Page, userid: UserId) => {
  * - Post-conditions: If `realmType` is `user`: User realm created, user
  *   is on the page that Tobira forwards to immediately after creating the realm.
  *   Otherwise, if `realmType` is `regular`, nothing was created but the user
- *   is on a non-root realm page ("/support").
+ *   is on a non-root realm page ("/support" or "/empty").
  */
 export const realmSetup = async (
     page: Page,
     userid: UserId,
     realmType: Realm,
     parentPageName: string,
+    empty?: boolean,
 ) => {
     await test.step("Setup", async () => {
         await page.goto("/");
         await login(page, userid);
 
         // Go to a non-root realm
         if (realmType === "Regular") {
-            await page.locator("nav").getByRole("link", { name: parentPageName }).click();
-            await expect(page).toHaveURL("/support");
+            if (empty) {
+                // This is an unfortunate workaround.
+                // The `nth()` locator used in block tests sometimes resolves to the wrong
+                // element if there is already a block present.
+                // I believe this is a bug in playwright. To prevent that from happening,
+                // the block tests should always be starting with an empty realm.
```

Comment:
Mh so I'd rather investigate this than adding a realm into the fixture and another parameter and branch to this method. Should be possible to solve this somehow differently. If you want, I can also take a look at this.

Reply:
Of course that would be preferable. You are welcome to take a look. It's not like I didn't try to find out what is causing this, but... no idea. Sometimes the `nth(3)` locator just resolves to the 5th element.

Reply:
Did some more testing and this is only sporadically happening when I'm running the test in a browser (i.e. not headless). So I guess I'll just revert this and we'll cross our fingers and hope it won't happen in CI 🤷

Reply:
This was indeed related to the `nth()` locator, but I think I found a solution that should work without feeling hacky.